ecosystem/gstreamer_plugin/gst_mtl_st40_rx.c

```diff
@@ -179,6 +181,47 @@ static void gst_mtl_st40_rx_class_init(Gst_Mtl_St40_RxClass* klass) {
                         G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 }
 
+static guint gst_mtl_st40_rx_parse_port_arguments(struct st40_rx_ops* ops_rx,
```

Comment:
why not use `gst_mtl_common_parse_rx_port_arguments()`?

Reply:
The ops_rx is disticly different we can't cast it either :< 
We need parsing function for st40_rx_ops* ops_rx, as we don't have the support for pipeline in RX